app/src/main/java/ar/scacchipa/twittercloneapp/di/SharedPrefs.kt

```diff
@@ -0,0 +1,11 @@
+package ar.scacchipa.twittercloneapp.di
```

Comment:
the name of this file should be better it is not a sharedpref it is a provider method to instruct di motor how to find one.